themes/sbvfrd/ts/carousel/DataLoader.ts

```diff
@@ -91,7 +91,7 @@ export default class DataLoader implements SearchResultProvider {
     private getSearchUrl(page: number, size: number): string {
         // TODO: Check whether the following is actually true
         // page is zero-based internally but one-based interpreted by server side
```

Comment:
Please can you remove that VuFind.path thing here again?
The carousel's configuration is generated in a .phtml file. I will change the path there by using the basePath helper

Reply:
Done